### PR TITLE
Runtime metrics and monitoring resolve #30

### LINKED ISSUE #30: Runtime metrics and monitoring
We would like to have some metrics of akubra runtime. Eg.:
Dla usprawnienia monitoringu akubry powinniśmy agregować i eksponować następujące metryki:
1) Number of requests per second
  - split by method
  - split by reponse status code
  - for each akubra's backend
2) Response times
  - split by method
  - split by reponse status code
  - for each akubra's backend
3) Number of non http errors
4) Amount of allocated memory
5) Number of running goroutines

### PR DESCRIPTION
**Pull request by mjarco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,21 @@ Logging:
...
+Metrics:
+  # Possible targets: "graphite", "expvar", "stdout"
+  Target: graphite
+  # Expvar handler listener address
+  ExpAddr: ":8080"
+  # How often metrics should be released, applicable for "graphite" and "stdout"
+  Interval: 20s
+  # Graphite metrics prefix path
+  Prefix: my.metrics
+  # Shall prefix shall be suffixed with "<hostname>.<process>"
```
Comment (ksitak) [inline README.md]: one "shall" too much

Comment (mjarco) [inline README.md]: :)

File: metrics/system_metrics.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+	"github.com/rcrowley/go-metrics"
+)
+
+const allocGauge = "runtime.mem.bytes_allocated_and_not_yet_freed"
+const heapObjectsGauge = "runtime.mem.total_number_of_allocated_objects"
+const totalPauseGauge = "runtime.mem.pause_total_ns"
+const lastPauseGauge = "runtime.mem.last_pause"
+
+func collectSystemMetrics() (err error) {
```
Comment (janisz) [inline metrics/system_metrics.go]: You can use https://godoc.org/github.com/rcrowley/go-metrics#RegisterRuntimeMemStats instead of this

Comment (mjarco) [inline metrics/system_metrics.go]: I like that compact set of most meaningful metrics
@ksitak @bieli You find this metics useful on daily basis? https://golang.org/pkg/runtime/#MemStats

Comment (mjarco) [inline metrics/system_metrics.go]: All in all, we will collect MemStats if there is Debug = true in Config

File: metrics/runtime_metrics.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+import (
+	"runtime"
+
+	"github.com/rcrowley/go-metrics"
+)
+
+const goroutinesNumGauge = "runtime.goroutines_num"
+
+func collectRuntimeMetrics() error {
```
Comment (mjarco) [inline metrics/runtime_metrics.go]: I don't see such need. Do you? Any use case?

**Pull request merged**